CONTRIBUTING.md

```diff
@@ -2,11 +2,17 @@
 
 ## **Did you find a bug?**
 
+Please file an issue [here](https://github.com/caos/zitadel/issues/new/choose).
+
 ## **Want to contribute code?**
 
-* Check out our [Dev Build Guide](build/README.md).
+### build for local development
 
-## **Did you find a security flaw?**
+* Check out our [Dev Build Guide](build/README.md#-Start-with-docker-compose).
 
-* Please read [Security Policy](SECURITY.md).
+### docker build
```

Comment:
Can we get a more self explaining title here?

Something like `Build locally with docker`

Reply:
changed the titles